packages/expo-cli/src/commands/fetch/android.ts

```diff
@@ -36,6 +36,7 @@ export async function fetchAndroidKeystoreAsync(
   const ctx = new Context();
   await ctx.init(projectDir, {
     nonInteractive: options.parent?.nonInteractive,
+    skipCredentialsCheck: false,
```

Comment:
Does this mean that if someone is running `fetch:android:keystore` and the keystore stored on our servers is invalid - we won't save it to the user's disk and just throw an error? Please correct me if I'm wrong.
I think this is not expected behavior and we shouldn't perform validation in this case (and other cases in this file). @wkozyra95 ?

Reply:
I think that in this context, this flag is meaningless. It's not triggering any code that supports it( I assume `skipCredentialsCheck: false` was added just for the sake of typescript)

I'm not really convinced that it should be part of the context

The expected behaviour, in my opinion, would be to check keystore only on create or update and only in that context skip-credentials-check make sense